### PR TITLE
Secondary button

### PR DESCRIPTION
**Pull request by ewal**
- based on link-button but with no link behavior
- used for cancelations etc.

screen shot 2015-03-04 at 13 08 39

### CODE REVIEW

File: src/mnd-bootstrap/css/_buttons.scss [hunk 1]
```diff
@@ -29,6 +30,15 @@
     @extend .btn-danger;
   }
 
+  &-secondary {
+    @extend .btn-link;
+    color: $text-color;
+    &:hover {
+      text-decoration: none;
+      color: #333;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_buttons.scss]: Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: src/mnd-bootstrap/css/_buttons.scss [hunk 2]
```diff
@@ -29,6 +30,15 @@
     @extend .btn-danger;
   }
 
+  &-secondary {
+    @extend .btn-link;
+    color: $text-color;
+    &:hover {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_buttons.scss]: Rule declaration should be preceded by an empty line

File: src/mnd-bootstrap/css/_buttons.scss [hunk 3]
```diff
@@ -29,6 +30,15 @@
...
+      text-decoration: none;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_buttons.scss]: Properties should be ordered color, text-decoration

**Comment by alexwoll** [comment]
Looks good!

**Pull request merged**